flow360/plugins/report/uvf_shutter.py

```diff
@@ -572,56 +573,79 @@ class Shutter(Flow360BaseModel):
     access_token: Optional[str] = None
 
     async def _get_3d_images(self, screenshots: dict[str, Tuple]) -> dict[str, list]:
-        @backoff.on_exception(lambda: backoff.constant(3), Flow360WebNotAvailableError, max_time=3600)
-        @http_interceptor
-        async def _get_image_sequence(
-            session: aiohttp.client.ClientSession, url: str, shutter_request: list[dict]
-        ) -> str:
-            log.debug(
-                f"sending request to uvf-shutter: {url=}, {type(shutter_request)=}, {len(json.dumps(shutter_request))=}"
-            )
-            return session.post(url, json=shutter_request)
-
-        async with aiohttp.ClientSession(
-            timeout=aiohttp.ClientTimeout(total=3600),
-            headers={"Authorization": f"Bearer {self.access_token}"},
-        ) as session:
-            tasks = []
-            for _, _, shutter_request in screenshots:
-                tasks.append(
-                    _get_image_sequence(
-                        session=session,
-                        url=urljoin(self.url, "/sequence/run"),
-                        shutter_request=shutter_request,
-                    )
-                )
-                log.debug(
-                    f"request to shutter: {json.dumps(shutter_request)}"
+
+        async def _build_image_from_local(uvf_request: list[dict], output_folder: str) -> list[str]:
+            input_folder = os.path.join(output_folder, 'input')
+            os.makedirs(input_folder, exist_ok=True)
+
+            log.info(f"dumping the uvf request {uvf_request} to local folder {input_folder}/sequence.json")
```

Comment:
Small nit pick but we should call it shutter where ever we can.  We had a whole thing about renaming the repos because they had UVF